packs/github/sensors/github_repository_sensor.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, sensor_service, config=None, poll_interval=None):
         self._last_event_ids = {}
 
     def setup(self):
-        self._client = Github(self._config['token'])
+        self._client = Github(self._config['token'] or None)
```

Comment:
Actually the better way to write this would've been `self._config.get('token', None)` to avoid any exceptions.

Since I've merged the change already I will pick-up the fix.

(note to self - coffee before work is essential)